File: JAVA/Daily Flash Codes/Week2/Day3_Jan22/code/prog4.java


import java.io.*;

class Program {
	public static void main(String[] args) throws IOException {
		BufferedReader in = new BufferedReader(new InputStreamReader(System.in));		//obj of Bufferedreader to take input
		
		System.out.println("Enter two Nos: ");
		int num1 = Integer.parseInt(in.readLine());		
		int num2 = Integer.parseInt(in.readLine());		

		System.out.println("Enter operation(+ , - , * , /)");
		char opcode = (char)in.read();
		switch(opcode) {
		case '+':
			System.out.println("Addition of " + num1 + "& " + num2 + ": "+ (num1+num2) );
			break;

		case '-':
			System.out.println("Substraction of " + num1 + "& " + num2 + ": "+ (num1-num2) );
			break;
		case '*':
			System.out.println("Multiplication of " + num1 + "& " + num2 + ": "+ (num1*num2) );
			break;
		case '/':
			System.out.println("Division of " + num1 + "& " + num2 + ": "+ (num1/num2) );
			break;
		default:
			System.out.println("Wrong opcode entered!!!" );
			break;

		}

	

	}


}

File: Number Series/Basic/prog4.c
#include <stdio.h>

void main() {
	int num1,num2;
	char opcode;

	//enter num1
	printf("Enter number 1: ");
	scanf("%d",&num1);

	//enter num2
	printf("Enter number 2: ");
	scanf("%d",&num2);

	//accpet opcode;
	printf("Enter operation(+,-,/,*): " );
	scanf(" %c",&opcode);

	switch(opcode) {
		case '+':
			printf("Addition of %d & %d : %d\n",num1,num2,num1+num2);
			break;

		case '-':
			printf("Substraction of %d & %d : %d\n",num1,num2,num1-num2);
			break;
		case '*':
			printf("Multiplication of %d & %d : %d\n",num1,num2,num1*num2);
			break;
		case '/':
			printf("Division of %d & %d : %d\n",num1,num2,num1/num2);
			break;
		
		}
}
